CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ if( BUILD_TESTING )
       set( ARCH_FLAG "" )
     endif()
 
-    set( REGRESSION_BASELINE_SHA "" CACHE STRING "SHA of baseline comparison" )
+    set( REGRESSION_BASELINE_SHA "73377e2" CACHE STRING "SHA of baseline comparison" )
```

Comment:
@Myoldmopar Did I do this right?  I want to compare this branch  with branch [ResetNodeDataEarlier](https://github.com/NREL/EnergyPlus/tree/ResetNodeDataEarlier)